### PR TITLE
Piglatin_translator scripts

### LINKED ISSUE #916: Pig Latin Translater
# Details
What the features will your script have

- This Python script converts a English sentence to a pig latin secret message

Please assign this to me as a part of gssoc'21

### PR DESCRIPTION
**Pull request by archanagandhi**
# Description
Scripts and README file has been added as per guidelines
## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@archanagandhi translation of `my` should be `ymay` but your code gives `myay`

**Comment by santushtisharma10** [comment]
image

@archanagandhi  fix it

**Comment by archanagandhi** [comment]
Hey @santushtisharma10, there is no pig latin for single letter consonants. The only single letter word in english are 'A' and 'I' only. You have used 'S' between the sentence so it shows error.

**Review by santushtisharma10 — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

File: Piglatin_Translator/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Pig Latin Translator
+A secret language formed from English by transferring the initial consonant or consonant cluster of each word to the end of the word and adding a vocalic syllable (usually /eɪ/): so pig Latin would be igpay atinlay
+
+# Rules
```
Comment (kaustubhgupta) [inline Piglatin_Translator/README.md:4]:
```suggestion
## Rules
```

File: Piglatin_Translator/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+1. If a word begins with a vowel, just as "yay" to the end. For example, "out" is translated into "outyay".
+2. If it begins with a consonant, then we take all consonants before the first vowel and we put them on the end of the word. For example, "which" is translated into "ichwhay".
+  
+# Input
```
Comment (kaustubhgupta) [inline Piglatin_Translator/README.md:8]:
```suggestion
## How to run?
```

File: Piglatin_Translator/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ 
+ - Run the script ```piglatin.py```
```
Comment (kaustubhgupta) [inline Piglatin_Translator/README.md:10]:
```suggestion
 - Run the script ```python piglatin.py```
```

File: Piglatin_Translator/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ - Enter a Sentence you need to translate to pig latin
+  
+ Example sentence: hello fellas my name is Archana
+   
+ # Output
```
Comment (kaustubhgupta) [inline Piglatin_Translator/README.md:15]:
```suggestion
## Output
```

File: Piglatin_Translator/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+![pigoutput](https://user-images.githubusercontent.com/74424757/115106340-dfd7ee00-9f81-11eb-8412-ad344823be1d.PNG)
+
+
+## [Author](https://github.com/archanagandhi)
```
Comment (kaustubhgupta) [inline Piglatin_Translator/README.md:19]:
```suggestion
## Author
```

File: Piglatin_Translator/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+Archana Gandhi
```
Comment (kaustubhgupta) [inline Piglatin_Translator/README.md:20]:
```suggestion
[Archana Gandhi](https://github.com/archanagandhi)
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**